### PR TITLE
Feature/tl2023 217

### PR DESCRIPTION
**Pull request by Skumarhpa**

### CODE REVIEW

File: src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/Assessment/AdminCoreComponentViewModel.cs [hunk 1]
```diff
@@ -11,7 +13,9 @@ public class AdminCoreComponentViewModel : AdminAssessmentLearnerDetails
     {
         public AdminAssessmentDetailsViewModel AssessmentDetails { get; set; }
 
-        public IEnumerable<PathwayAssessments> ValidPathwayAssessmentSeries { get; set; }
+        public IEnumerable<PathwayAssessments> PathwayAssessments { get; set; }
+
+        public List<PathwayAssessments> ValidPathwayAssessmentSeries { get; set; }
```
Comment (sajidmalik) [inline src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/Assessment/AdminCoreComponentViewModel.cs:18]: Please update this to IEnumerable. I have made this change in my branch.

Comment (Skumarhpa) [reply src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/Assessment/AdminCoreComponentViewModel.cs]: Done the change

File: src/Tests/Sfa.Tl.ResultsAndCertification.Web.UnitTests/Controllers/AdminDashboardControllerTests/AdminReviewChangeCoreAssessmentGet/When_Called_With_Invalid_Data.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+using Sfa.Tl.ResultsAndCertification.Web.Loader;
+using Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard.Assessment;
+using Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard.IndustryPlacement;
+using System.Threading.Tasks;
+using Xunit;
+namespace Sfa.Tl.ResultsAndCertification.Web.UnitTests.Controllers.AdminDashboardControllerTests.AdminReviewChangeCoreAssessmentGet
+{
+    public class When_Called_With_Invalid_Data: TestSetup
+    {
+        protected AdminReviewChangesCoreAssessmentViewModel Mockresult = null;
```
Comment (sajidmalik) [inline src/Tests/Sfa.Tl.ResultsAndCertification.Web.UnitTests/Controllers/AdminDashboardControllerTests/AdminReviewChangeCoreAssessmentGet/When_Called_With_Invalid_Data.cs:15]: Is this being used?

Comment (Skumarhpa) [reply src/Tests/Sfa.Tl.ResultsAndCertification.Web.UnitTests/Controllers/AdminDashboardControllerTests/AdminReviewChangeCoreAssessmentGet/When_Called_With_Invalid_Data.cs:15]: Yes

**Review by sajidmalik — APPROVED** [review]

**Pull request merged**